WcaOnRails/app/models/registration.rb

```diff
@@ -243,6 +243,16 @@ def self.wcif_json_schema
     end
   end
 
+  validate :must_register_for_lte_event_limit
```

Comment:
Can you give this a better name? Not sure what lte is supposed to mean

Reply:
Thanks - fixed.